frontend/src/components/widgets/TextArea/TextArea.tsx

```diff
@@ -208,6 +208,7 @@ class TextArea extends React.PureComponent<Props, State> {
           <UITextArea
             data-testid="stTextArea"
             value={value}
+            clearable={element.clearable}
```

Comment:
nit: I think we can also set `clearOnEscape={element.clearable}` here as well.

Reply:
True, it's updated, same as above, thank you 👍